std_msgs/Header header
uint8[5] bytes
uint8 data_len
uint32 can_id